packages/@glimmer/syntax/lib/get-template-locals.ts

```diff
@@ -56,7 +74,7 @@ function addTokens(tokensSet: Set<string>, node: ASTv1.Node, scopedTokens: strin
  * referenced that could possible come from the praent scope. Can exclude known keywords
  * optionally.
  */
-export function getTemplateLocals(html: string, options?: { includeKeywords: boolean }): string[] {
+export function getTemplateLocals(html: string, options: GetTemplateLocalsOptions = {}): string[] {
```

Comment:
I think we should default both of the options, something like:

```suggestion
export function getTemplateLocals(html: string, options: GetTemplateLocalsOptions = { includeKeywords: false, includeHtmlElements: false }): string[] {
```

I believe this will be the common case.